### PR TITLE
add minio multi-arch image and drone build setup

### PR DESCRIPTION
**Pull request by meldafrawi**
Signed-off-by: Mohamed Eldafrawi <mohamed.eldafrawi@rancher.com>

### CODE REVIEW

**Comment by meldafrawi** [comment]
@yasker can you please check this PR ?

File: .drone.yml [hunk 1]
```diff
@@ -0,0 +1,303 @@
...
+    custom_dns: 1.1.1.1
+    dry_run: True
+    dockerfile: Dockerfile.s390x
+    repo: longhornio/minio
+  when:
+    branch:
+    - main
+---
+kind: pipeline
+name: pulish
```
Comment (yasker) [inline .drone.yml:101]: `publish`.

File: .drone.yml [hunk 2]
```diff
@@ -0,0 +1,303 @@
...
+  - ./export-tags.sh amd64
+  when:
+    instance:
+    - drone-publish.longhorn.io
+    branch:
+    - main
+    event:
+    - push
+
+- name: pusblish-amd64
```
Comment (yasker) [inline .drone.yml:134]: typo.

**Review by yasker — COMMENTED** [review]
Can you add a README.md to explain how does this work?

Also, I saw the event is `push`, which means we need to do something for every upstream update? I am thinking if we can just let drone periodically run, e.g. every day to check for necessary releases.

File: .drone.yml (hunk 1 above)
Comment (meldafrawi) [reply .drone.yml]: fixed.

File: .drone.yml (hunk 2 above)
Comment (meldafrawi) [reply .drone.yml]: fixed.

**Comment by meldafrawi** [comment]
@yasker for running drone periodically, settings is set through drone server UI.

**Review by yasker — APPROVED** [review]

**Comment by meldafrawi** [comment]
Ref: https://github.com/longhorn/longhorn/issues/1887

**Pull request merged**